frontend/src/css/PublicProfile.styl

```diff
@@ -53,6 +53,13 @@
       background-color veryLightGrey
       height 200px
       margin 10px
+  &__customForm__submit
+    position absolute
+    top 0
+    right 0
+    padding 5px 8px
+    font-size 18px
```

Comment:
Don't we have a `fontSize` const at frontend_lib?

Reply:
ok, will use fontSizeMedium

Reply:
or add a `fontSizeDefault` I thought we had

Reply:
20px is fine and since it uses an already used variable, it's more consistent than 18. So I'm fine with it

Reply:
fixed